.github/workflows/build_and_test.yml

```diff
@@ -15,17 +15,29 @@ jobs:
             python: xvfb-run python3
             pip_arg: ""
             nosetests_args: --with-coverage --cover-package epregressions  # --cover-html for local html results
-            build_arg: Ubuntu20
+            pyinstall_args: ""
+            add_data: "epregressions/diffs/math_diff.config:epregressions/diffs"
+            pack_tool: tar -zcvf
+            extension: .tar.gz
+            pack_suffix: -C dist runner
```

Comment:
Pulled some new args into the workflow matrix.  I wish I could collapse them down a little, but I think this is fine.  I don't anticipate it getting much bigger.